CHANGELOG.md

```diff
@@ -58,7 +58,7 @@
 - **Prometheus Scaler:** Support for `X-Scope-OrgID` header ([#2667](https://github.com/kedacore/keda/issues/2667))
 - **RabbitMQ Scaler:** Include `vhost` for RabbitMQ when retrieving queue info with `useRegex` ([#2498](https://github.com/kedacore/keda/issues/2498))
 - **Selenium Grid Scaler:** Consider `maxSession` grid info when scaling. ([#2618](https://github.com/kedacore/keda/issues/2618))
-
+- **TriggerAuthentication:** Provide support for specifying identity to use for Azure managed identity auth ([#2656](https://github.com/kedacore/keda/issues/2656))
```

Comment:
I'd move mark this as **General:**, but let's see what others think. @kedacore/keda-contributors

Reply:
Authentication and General are both fine for me

Reply:
So, what do we do? Let's not wait too long as I'd prefer not to have to resolve dozens of conflicts again...the more we wait, the more likely. 

Thanks

Reply:
Let's use general

Reply:
Ok moved to general